coding-tracker/coding-tracker/Controller.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System.Configuration;
+using Microsoft.Data.Sqlite;
+using Dapper;
+
+namespace coding_tracker
+{
+    internal class Controller
+    {
+        string connectionString = ConfigurationManager.AppSettings.Get("ConnectionString") + ConfigurationManager.AppSettings.Get("DatabasePath");
+
+        public void InitialiseDatabase()
+        {
```

Comment:
🔴 **Not Dapper**

💡 Change this method to use Dapper. Reach out if you need support.